source/errors.rst

```diff
@@ -72,7 +72,7 @@ which does not have a valid project.)
 
   {
     "status": 409,
-    "error": "Invalid foreign key",
+    "error": "Invalid Foreign Key",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L58 shows `'Invalid foreign key'` is the correct form.